tests/cms/views/poi/test_poi_form.py

```diff
@@ -167,8 +167,8 @@ def test_poi_in_use_not_deleted(
             == f"{settings.LOGIN_URL}?next={delete_poi}"
         )
     elif role in HIGH_PRIV_STAFF_ROLES:
-        with pytest.raises(ProtectedError):
-            response = client.post(delete_poi)
+        response = client.post(delete_poi)
+        assert poi_id is not None
```

Comment:
Even if the poi would have been deleted, poi_id would not change, since it is never updated